development/cpp/configuring_an_ide/clion.rst

```diff
@@ -3,69 +3,80 @@
 CLion
 =====
 
-`CLion <https://www.jetbrains.com/clion/>`_ is a commercial
+`CLion <https://www.jetbrains.com/clion/>`_ is a commercial 
 `JetBrains <https://www.jetbrains.com/>`_ IDE for C++.
 
 Importing the project
----------------------
+---------------------------------------------------
```

Comment:
reStructuredText **requires** titles underlines to be the same length as the titles themselves:

```suggestion
---------------------
```